eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -66,10 +66,12 @@ jobs:
         displayName: "Use Python 3.6"
         inputs:
           versionSpec: "3.6"
-      - template: /eng/common/pipelines/templates/steps/verify-links.yml
-        parameters:
-          Directory: sdk/${{ parameters.ServiceDirectory }}
-          CheckLinkGuidance: $true
+      - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+        - template: /eng/common/pipelines/templates/steps/verify-links.yml
+          parameters:
+            Directory: ''
+            Urls: (git diff origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH} HEAD --name-only -- '*.md')
```

Comment:
Can't we encapsulate this logic inside `/eng/common/pipelines/templates/steps/verify-links.yml` ?

Reply:
We currently don't want to update all lang repo. Some languages repos do not have aggregate-report pipeline setup. So it is less the ability of scanning the entire repo. We only update 4 major langs for now. Once we want to update to all repo, we will move the logic inside of verify-links.yml